app/utils/RetryHelper.scala

```diff
@@ -49,7 +49,7 @@ trait RetryHelper  {
           val call: Callable[Future[Int]] = new Callable[Future[Int]](){
             def call(): Future[Int] = Future.successful(1)
           }
-          logger.warn(s"Failure, retrying after $wait ms, attempt $currentAttempt")
+          logger.info(s"Failure, retrying after $wait ms, attempt $currentAttempt")
```

Comment:
This might be useful, so people know that initial requests failed. Should be warn.